RNTester/RNTesterPods.xcodeproj/project.pbxproj

```diff
@@ -17,7 +17,9 @@
 		3D13F84A1D6F6AFD00E69E0E /* OtherImages.xcassets in Resources */ = {isa = PBXBuildFile; fileRef = 3D13F8451D6F6AF200E69E0E /* OtherImages.xcassets */; };
```

Comment:
unrelated: I thought we're not supposed to check in .xcodeproj/.xcworkspace file when using cocoapods? should probably remove this in different PR

Reply:
Generally you check in the `xcodeproj` but not the `xcworkspace` when working with pods.

Reply:
ok sounds good